syntaxes/d2.tmLanguage.json

```diff
@@ -285,12 +285,12 @@
         },
         {
           "name": "keyword.operator.ampersand.d2",
-          "match": "&"
+          "match": "\u0026"
```

Comment:
huh, why did these change?

Reply:
Must be the version of yq I'm using. I'll investigate. Its equivalent FWIW.